packages/react-components/react-tabster/src/hooks/useArrowNavigationGroup.ts

```diff
@@ -25,14 +25,27 @@ export interface UseArrowNavigationGroupOptions {
    * Tabster should ignore default handling of keydown events
    */
   ignoreDefaultKeydown?: Types.FocusableProps['ignoreKeydown'];
+  /**
+   * The default focusable item in the group will be an element with Focusable.isDefault property
```

Comment:
can you add that currently there is no way in react-tabster to set isDefault and that this option is for internal testing purposes?